src/Accounts/Authentication/Factories/ClientFactory.cs

```diff
@@ -67,7 +67,15 @@ public ClientFactory()
             var subscriptionId = typeof(TClient).GetProperty("SubscriptionId");
             if (subscriptionId != null && context.Subscription != null)
             {
-                subscriptionId.SetValue(client, context.Subscription.Id.ToString());
+                // TODO: Once subscription type is updated to Guid everywhere, will no longer need this check.
```

Comment:
We just discussed this modification. We understand that this is caused by a type change in common type v5 of swagger, and need to be resolved because other modules will also face same issues. But as it's a change in a base class, we need to investigate whether this would have deeper impact.

The investigation would cost some time, and this PR may not catch current release. If you are in a hurry, we might consider update the common type version in swagger using directive first.